### PR TITLE
HwF-AWP-Processing-Cron-Job

### PR DESCRIPTION
**Pull request by sahaSonali**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/PRL-6026

### Change description ###
Added a new cron job for citizen AWP help with fees processing.

## 🤖AEP PR SUMMARY🤖

_I'm a bot that generates AI summaries of pull requests, see [AEP](https://kainossoftwareltd.github.io/ai-enhanced-platform/) for more details_

- apps/private-law/base/kustomization.yaml
  - Added a new resource prl-cron-hwf-process-awp.yaml to the kustomization file.

- apps/private-law/demo/base/kustomization.yaml
  - Added a new path to the demo.yaml from prl-cron-hwf-process-awp.

- apps/private-law/prl-cron-hwf-process-awp/aat.yaml
  - Added a new HelmRelease resource aat.yaml for prl-cron-hwf-process-awp.

- apps/private-law/prl-cron-hwf-process-awp/demo-image-policy.yaml
  - Added a new ImagePolicy resource demo-image-policy.yaml for prl-cron-hwf-process-awp.

- apps/private-law/prl-cron-hwf-process-awp/demo.yaml
  - Added a new HelmRelease resource demo.yaml for prl-cron-hwf-process-awp.

- apps/private-law/prl-cron-hwf-process-awp/ithc.yaml
  - Added a new HelmRelease resource ithc.yaml for prl-cron-hwf-process-awp.

- apps/private-law/prl-cron-hwf-process-awp/perftest.yaml
  - Added a new HelmRelease resource perftest.yaml for prl-cron-hwf-process-awp.

- apps/private-law/prl-cron-hwf-process-awp/prl-cron-hwf-process-awp.yaml
  - Added a new HelmRelease resource prl-cron-hwf-process-awp.yaml for prl-cron-hwf-process-awp.

- apps/private-law/prl-cron-hwf-process-awp/prod.yaml
  - Added a new HelmRelease resource prod.yaml for prl-cron-hwf-process-awp.

### CODE REVIEW

File: apps/private-law/prl-cron-hwf-process-awp/demo.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+metadata:
+  name: prl-cron-hwf-process-awp
+  namespace: private-law
+spec:
+  releaseName: prl-cron-hwf-process-awp
+  values:
+    job:
+      image: hmctspublic.azurecr.io/prl/cos:prod-5ffc34f-20240524155420 #{"$imagepolicy": "flux-system:demo-prl-cron-hwf-process-awp"}
+      environment:
+        TASK_NAME: Fm5ReminderNotificationTask
```
Comment (yogendraU) [inline apps/private-law/prl-cron-hwf-process-awp/demo.yaml:12]: please update the task name

File: apps/private-law/prl-cron-hwf-process-awp/demo.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        HEARING_AWAY_DAYS: 18
```
Comment (yogendraU) [inline apps/private-law/prl-cron-hwf-process-awp/demo.yaml:13]: please remove this env var

File: apps/private-law/prl-cron-hwf-process-awp/prl-cron-hwf-process-awp.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+metadata:
+  name: prl-cron-hwf-process-awp
+  namespace: private-law
+spec:
+  releaseName: prl-cron-hwf-process-awp
+  values:
+    job:
+      image: hmctspublic.azurecr.io/prl/cos:prod-09313a2-20240702154357 #{"$imagepolicy": "flux-system:prl-cos"}
+      environment:
+        TASK_NAME: Fm5ReminderNotificationTask
```
Comment (yogendraU) [inline apps/private-law/prl-cron-hwf-process-awp/prl-cron-hwf-process-awp.yaml:12]: rename the task

File: apps/private-law/prl-cron-hwf-process-awp/prl-cron-hwf-process-awp.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        HEARING_AWAY_DAYS: 18
+        VAR: trigger1
+        FEATURE_EXAMPLE: true
+        APP_ENV: "prod"
+        IDAM_API_URL: https://idam-api.platform.hmcts.net
+        AUTH_IDAM_CLIENT_BASEURL: https://idam-api.platform.hmcts.net
+        XUI_URL: https://manage-case.platform.hmcts.net/cases/case-details
+        CITIZEN_URL: www.apply-to-court-about-child-arrangements-c100.service.gov.uk
+        FM5_REMINDER_APPLICANT_RESPONDENT_SOLICITOR: d-b6b04932b1ac464b9a8c718d1b1a604a
```
Comment (yogendraU) [inline apps/private-law/prl-cron-hwf-process-awp/prl-cron-hwf-process-awp.yaml:21]: we may not need this env var

**Comment by sahaSonali** [comment]
Reopen

**Review by willwatters — APPROVED** [review]

**Pull request merged**